packages/siop-oid4vp/lib/types/SIOP.types.ts

```diff
@@ -179,6 +179,7 @@ export interface AuthorizationResponsePayload {
     | MdocOid4vpMdocVpToken
   presentation_submission?: PresentationSubmission
   verifiedData?: IPresentation | AdditionalClaims
+  is_first_party?: boolean
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   [x: string]: any
```

Comment:
```suggestion
  [x: string]: unknown
```

Reply:
Please don't make these types of changes at present. It potentially has a cascading effect. If you have external code that has subproperties it will now fail. And to be honest for a public api like this that is unworkable.  The whole idea is that we can have any object with sub objects in there. We don't care and don't use it to begin with. Now all of a sudden implementorts using our public API would be impacted.